monitor/monitor.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python3
+"""
+
+This is a quick hack and adopted plugin from the summary.py plugin (orinigally written by Rusty Russell
+
+This one is adapted by Rene Pickhardt and aimed to help you identify inactive channels quickly
+
+"""
+
+from lightning import Plugin, Millisatoshi
+import lightning
+import json
+
+plugin = Plugin(autopatch=True)
```

Comment:
`autopatch=True` is now the default, so no need to specify it all the time :-)
It uses the `LIGHTNINGD_PLUGIN` environment variable to decide whether we are running as a plugin or not.

Reply:
good to know. will leave it here to save another git commit